.circleci/config.yml

```diff
@@ -30,16 +30,21 @@ jobs:
             fi
   build_and_test_and_deploy_ui:
     docker:
-      - image: allofustest/workbench:buildimage-0.0.4
+      - image: allofustest/workbench:buildimage-0.0.5
     working_directory: ~/workbench
     environment: []
     steps:
       - checkout
       - run:
-          working_directory: ~/
-          name: Set up swagger CLI
+          working_directory: ~/workbench/ui
+          name: Fetch submodules
+          command: |
+            git submodule update --init libproject/utils
```

Comment:
FYI, I think running `update --init` without a path works, and keeps us from having to coordinate changes to the submodule. (But I'm happy if you prefer to specify the path.)